ledger/internal/eval.go

```diff
@@ -1217,7 +1428,7 @@ func (eval *BlockEvaluator) resetExpiredOnlineAccountsParticipationKeys() error
 		acctData.ClearOnlineState()
```

Comment:
looks like we have two `ClearOnlineState` function one for `basic.AccountData` and the other for `ledgercore.AccountData` and the first is not referenced  if I understand correctly

Reply:
yes - my bad, a merge mistake..